core-ui/src/shared/components/StatsGraph/StatsPanel.js

```diff
@@ -179,23 +221,23 @@ export function StatsPanel({ type, mode = 'single', ...props }) {
     return '';
   }
 
-  const graphOptions =
-    type === 'pod'
-      ? ['cpu', 'memory', 'network']
-      : ['cpu', 'memory', 'network', 'nodes'];
-
+  const graphOptions = getGraphOptions(type);
   return (
     <LayoutPanel className="fd-margin--md stats-panel">
       <LayoutPanel.Header>
         <LayoutPanel.Filters>
-          <Dropdown
-            selectedKey={metric}
-            onSelect={(e, val) => setMetric(val.key)}
-            options={graphOptions.map(option => ({
-              key: option,
-              text: t(`graphs.${option}`),
-            }))}
-          />
+          {props?.title ? (
```

Comment:
Maybe we could just add an option for PVC to the `getGraphOptions`, and then here show a title if there is only 1 option, and a dropdown if there are multiple? Also, we could instead of passing the title from the parent component, use the title same approach as we do for the dropdown `t('graphs.${option}')`.